File: AR-VR/arshivaastha_Astha_2024cse1096_2/IntroductionToRecursion/PrintIncreasing.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws Exception {
        // write your code here
        Scanner in= new Scanner (System.in);
        int n= in.nextInt();
        printIncreasing(n); 
    }

    public static void printIncreasing(int n){
           if(n==0)
        return;
          printIncreasing(n-1);
          System.out.println(n);
        
    }

}

File: Web/Prakhar-08_PrakharRastogi_1923it1011_3/Introduction to Recursion/Print Increasing.cpp
#include<iostream>
using namespace std;

void printIncreasing(int n){
    // write your code here
    if(n == 1)
    {
        cout << 1 << endl;
        return;
    }
    
    printIncreasing(n-1);
    cout << n << endl;
    
}


int main(){
    int n; cin>>n;
    printIncreasing(n);
}
